partials/confirm_login.html

```diff
@@ -0,0 +1,51 @@
+  </td>
+</tr>
+
+<!--Button Area-->
+<tr height="50">
+  <td align="center" valign="top">
+    <table border="0" cellpadding="0" cellspacing="0" height="100%" width="100%" id="email-button" style="-webkit-text-size-adjust: 100%; border-collapse: collapse; mso-table-lspace: 0pt; mso-table-rspace: 0pt; background-color: #0095dd; border-radius: 4px; height: 50px; width: 310px !important;">
+      <tr style="page-break-before: always">
+        <td align="center" valign="middle" id="button-content" style="font-family: sans-serif; font-weight: normal; text-align: center; margin: 0; color: #ffffff; font-size: 20px; line-height: 100%;">
+          <!--[if mso]>
+          <v:roundrect xmlns:v="urn:schemas-microsoft-com:vml" xmlns:w="urn:schemas-microsoft-com:office:word" href="{{{link}}}" style="width:280px;height:40px;v-text-anchor:middle;" arcsize="10%" stroke="f" fillcolor="#0095DD">
+            <w:anchorlock/>
+            <center>
+          <![endif]-->
+          <a href="{{{link}}}" id="button-link" style="font-family:sans-serif; color: #fff; display: block; padding: 15px; text-decoration: none; width: 280px;">{{t "Confirm sign-in"}}</a>
+          <!--[if mso]>
+          </center>
+          </v:roundrect>
+          <![endif]-->
+        </td>
+      </tr>
+    </table>
+  </td>
+</tr>
+<!--Button Area-->
+<tr style="page-break-before: always">
+  <td border="0" cellpadding="0" cellspacing="0" height="100%" width="100%">
+    <br/>
+    <p width="310" class="secondary" style="font-family: sans-serif; font-weight: normal; margin: 0 0 24px 0; text-align: center; color: #8A9BA8; font-size: 11px; line-height: 13px; width: 310px !important; word-wrap: break-word; word-break: break-all">{{t "Alternatively:"}}<a href="{{{link}}}" style="color: #0095dd; text-decoration: none; width: 310px !important; display:block;"><br/><font style="word-break:break-all;">{{{link}}}</font></a></p>
+    <p class="secondary" style="font-family: sans-serif; font-weight: normal; margin: 0; text-align: center; color: #8A9BA8; font-size: 11px; line-height: 13px; width: 310px !important; word-wrap:break-word">{{t "This is an automated email; if you received it in error, no action is required."}} {{{t "For more information, please visit <a %(supportLinkAttributes)s>Mozilla Support</a>"}}}</p>
+  </td>
+</tr>
+<% endblock %>
+
+<% block after_table %>
+<div itemscope itemtype="https://schema.org/EmailMessage">
+  <div itemprop="potentialAction" itemscope itemtype="https://schema.org/ViewAction">
+    <link itemprop="target" href="{{{oneClickLink}}}"/>
+    <meta itemprop="name" content="{{t 'Verify Email'}}"/>
+    <meta itemprop="url" content="{{{oneClickLink}}}"/>
+  </div>
+  <meta itemprop="description" content="{{t 'Verify your email to finish your Firefox Account registration'}}"/>
```

Comment:
@ryanfeeley - this is not for the email itself, this is for the future one click verification for Google.